### PR TITLE
Query update animals

### PR DESCRIPTION
**Pull request by ahzia**
Hello,
In this pull I have worked on:

- Added a columns species of type string to table ✔️
- Inserted new data for animals ✔️
- Created transaction file ✔️
- Updated animals table inside of transaction ✔️
- Deleted all animals born after Jan 1st, 2022 inside of transaction ✔️
- Write queries for new data ✔️
Kindly Review it and share your feedback with me.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -14,4 +14,33 @@ SELECT name FROM animals WHERE neutered IS TRUE;
...
 
-SELECT name FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT name FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals SET species='unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+
```
Comment (sinansevgi) [inline queries.sql:24]: - [x] You should verify that change was undone after rollback as requested in the project requirements

>Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and **verify that species columns went back to the state before tranasction**.

File: queries.sql [hunk 2]
```diff
@@ -14,4 +14,33 @@ SELECT name FROM animals WHERE neutered IS TRUE;
...
+BEGIN;
+UPDATE animals SET species='digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species='pokemon' WHERE species IS NULL;
+COMMIT;
```
Comment (sinansevgi) [inline queries.sql:28]: - [x] You should verify that change was done after commit as requested in the project requirements

> - Commit the transaction.
> - Verify that change was made and persists after commit.

File: queries.sql [hunk 3]
```diff
@@ -14,4 +14,33 @@ SELECT name FROM animals WHERE neutered IS TRUE;
...
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+
```
Comment (sinansevgi) [inline queries.sql]: - [x] You should verify that change was undone after rollback as requested in the project requirements

> - Now, take a deep breath and... Inside a transaction delete all records in the animals table, then roll back the transaction.
> - After the roll back verify if all records in the animals table still exist. After that you can start breath as usual ;)

**Review by sinansevgi — CHANGES_REQUESTED** [review]
# Changes Required :recycle: 
------

Hi @ahzia, 
Most of the queries work fine :+1: 
Good job so far!

There are some issues that you still need to work on to go further but you are almost there!



 ## Required Changes ♻️
_Please check the inline comments for my suggestions_ :arrow_down:

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi

------

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ahzia** [comment]
@sinansevgi  Thanks for your review, I implemented all required changes.

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @ahzia , 

I can see that you are verifying the changes  now :+1: 
Your project is complete!  
Congratulations! 🎉

You can merge this pull request and continue with your journey :wave: 


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ahzia** [comment]
@sinansevgi Thanks for the Review.

**Pull request merged**